frontend/app/adapters/user-human.js

```diff
@@ -1,9 +1,9 @@
 import ApplicationAdapter from "./application";
 
-export default ApplicationAdapter.extend({
+export default class UserHumanAdapter extends ApplicationAdapter {
   pathForType() {
     return "user_humen";
```

Comment:
user_human ?

Reply:
Nope, user_humen is correct. This refers to the [type attribute in json api](https://jsonapi.org/format/#document-resource-object-identification) which we always serialize pluralized. In this case, user_humen